src/world.cpp

```diff
@@ -972,7 +972,7 @@ int World_SetFlipState(uint32_t flip_index, uint32_t flip_state)
         return 0;
     }
 
-    if(global_world.flip_map[flip_index] == 0x1F)         // Check flipmap state.
+    if((global_world.flip_map[flip_index] == 0x1F) || (flip_state == 0))        // Check flipmap state.
```

Comment:
Switch trigger use flip_state=1 presently, so it doesn't work. FLIP_OFF is the only trigger that use it in OT.